runtime/browser/lib/auto-tabs.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
```

Comment:
Another bit of sloppiness. I was exploring the idea of layout objects that could support multiplexing slots. Autotabs in theory would create a tab interface allowing one to render multiple things into the root slot. Supporting multiple root targets turned out to be the hard part (involves SlotComposer refactor stuff we've been discussing) so I left it unfinished. 

I'd like to leave the code there, but I should have at least turned off the rendering.